extension/scripts/Selector/SelectorTable.js

```diff
@@ -158,19 +164,69 @@ var SelectorTable = {
         });
         return dataColumns;
     },
-
-    getFeatures: function () {
-        return ['selector', 'multiple', 'columns', 'delay', 'tableDataRowSelector', 'tableHeaderRowSelector', 'tableAddMissingColumns']
+    isExtractedDataColumns: function (header) {
+        answer = undefined;
+        this.columns.forEach(function (column) {
+            if (column.extract === true && header === column.header.trim()) {
+                answer = column.name;
+            }
+        });
+        return answer;
     },
 
-    getItemCSSSelector: function () {
-        return "table";
+
+
+    /**
+     * Extract table header column info from html
+     * @param html
+     */
+    getTableHeaderColumnsFromHTML: function (headerRowSelector, html, verticalTable) {
+        var $table = $(html);
+        var $headerRowColumns = $table.find(headerRowSelector);
+
+        if ($headerRowColumns.length > 1) {
```

Comment:
Later you use `$headerRowColumns.prevObject` for vertical tables, so I believe this block is intended only for horizontal tables. If so, wrap it under `if (!verticalTable) {...}` condition block and don't use `.prevObject`